### PR TITLE
docs: add test case blog

### PR DESCRIPTION
**Pull request by guitu168**
add test case blog.

Screenshots of the change:

### CODE REVIEW

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/06/27/getting-start-with-apisix-test-cases.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+以下为上述命令的正常执行结果。  
+
+![network error/normal return result](https://static.apiseven.com/2022/blog/0627/5.png)
+
+  如果测试失败，则会出现以下信息：
+
+![network error/fail return result](https://static.apiseven.com/2022/blog/0627/2.png)
+
+  以上信息则会告诉你具体是哪个测试文件中的哪个测试用例执行失败了。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/06/27/getting-start-with-apisix-test-cases.md]:
```suggestion
 如果测试失败，则会出现以下信息：

![network error/fail return result](https://static.apiseven.com/2022/blog/0627/2.png)

 以上信息则会告诉你具体是哪个测试文件中的哪个测试用例执行失败了。
```

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/06/27/getting-start-with-apisix-test-cases.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+
+![network error/Github CI example](https://static.apiseven.com/2022/blog/0627/3.png)
+
+我们可以看到，在该示例中，`416` 行出现了报错。通过错误信息，我们可以得到在某个测试文件中的某个测试用例出现错误，开发者定向查看修正即可。需要注意的是，CI 中可能存在一些奇怪的报错，它们可能是因为 CI 环境的临时异常导致的，如果未修改过对应模块中的代码，可以忽略这些错误。
+
+![network error/Github CI error example](https://static.apiseven.com/2022/blog/0627/4.png)
+
+## 总结
+
+本文主要为大家介绍了测试的相关流程，以及在 APISIX 测试案例的构成和如何进行测试案例的编写，希望通过本文你可以对 APISIX 的测试案例有一个大致的认识。然而本文只提到了 APISIX 测试框架中的一些核心内容，未能覆盖 `TEST::NGINX` 框架中的全部内容，实际上 `TEST::NGINX` 中还有很多强大的能力，我们可以通过 [`Test::Nginx::Socket` 的文档](https://metacpan.org/pod/Test::Nginx::Socket)了解更多用法。如果你想学习更多编写测试案例的知识，可以观看[APISIX测试案例快速入门视频](https://www.bilibili.com/video/BV1qF411G78j?spm_id_from=333.1007.top_right_bar_window_history.content.click&vd_source=d427acb1117c07aeea8968e15736e375)。
```
Comment (SylviaBABY) [inline website/i18n/zh/docusaurus-plugin-content-blog/2022/06/27/getting-start-with-apisix-test-cases.md]:
```suggestion
本文主要为大家介绍了测试的相关流程，以及在 APISIX 测试案例的构成和如何进行测试案例的编写，希望通过本文你可以对 APISIX 的测试案例有一个大致的认识。

然而本文只提到了 APISIX 测试框架中的一些核心内容，未能覆盖 `TEST::NGINX` 框架中的全部内容，实际上 `TEST::NGINX` 中还有很多强大的能力，我们可以通过 [`Test::Nginx::Socket` 的文档](https://metacpan.org/pod/Test::Nginx::Socket)了解更多用法。如果你想学习更多编写测试案例的知识，可以观看[APISIX 测试案例快速入门视频](https://www.bilibili.com/video/BV1qF411G78j?spm_id_from=333.1007.top_right_bar_window_history.content.click&vd_source=d427acb1117c07aeea8968e15736e375)。
```

File: website/blog/2022/06/27/getting-start-with-apisix-test-cases.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+
+![network error/Github CI error example](https://static.apiseven.com/2022/blog/0627/4.png)
+
+## Summary
+
+This article mainly introduces the relevant process of testing, as well as the composition of APISIX test cases and how to write test cases. I hope you can have a general understanding of APISIX test cases through this article. However, the text only mentions some core content in the APISIX test framework, and fails to cover all the content in the TEST::NGINX framework. In fact, there are many powerful capabilities in `TEST::NGINX`. We can use [Test::Nginx::Socket](https://metacpan.org/pod/Test::Nginx::Socket) documentation for more usage. If you want to learn more about writing test cases, you can also watch the [APISIX Test Cases Quick Start video].
```
Comment (SylviaBABY) [inline website/blog/2022/06/27/getting-start-with-apisix-test-cases.md]:
```suggestion
This article mainly introduces the relevant process of testing, as well as the composition of APISIX test cases and how to write test cases. I hope you can have a general understanding of APISIX test cases through this article. 

However, the text only mentions some core content in the APISIX test framework, and fails to cover all the content in the TEST::NGINX framework. In fact, there are many powerful capabilities in `TEST::NGINX`. We can use [Test::Nginx::Socket](https://metacpan.org/pod/Test::Nginx::Socket) documentation for more usage. If you want to learn more about writing test cases, you can also watch the [APISIX Test Cases Quick Start video](https://www.bilibili.com/video/BV1qF411G78j?spm_id_from=333.1007.top_right_bar_window_history.content.click&vd_source=d427acb1117c07aeea8968e15736e375).
```

Comment (SylviaBABY) [reply website/blog/2022/06/27/getting-start-with-apisix-test-cases.md]: forget the video link in the end

File: website/i18n/zh/docusaurus-plugin-content-blog/2022/06/27/getting-start-with-apisix-test-cases.md (hunk 2 above)
Comment (SylviaBABY) [reply website/i18n/zh/docusaurus-plugin-content-blog/2022/06/27/getting-start-with-apisix-test-cases.md]: [APISIX 测试案例快速入门视频]

**Review by SylviaBABY — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]
the URL is too long 😅

**Pull request merged**